app/controllers/newflow/login_signup_controller.rb

```diff
@@ -67,6 +67,12 @@ def educator_signup
       )
     end
 
+    def educator_complete_form
+      book_data = FetchBookData.new
```

Comment:
suggestion:  create a "factory" (private) method that creates this thru memoization.

```
private
def book_data
  @book_data ||= FetchBookData.new
end

```
 then use it as book_data.subjects, book_data.titles

Reply:
Ah, yes. Thank you for the suggestion!

Reply:
Great idea. Fixed.